packages/schematics/angular/service-worker/index_spec.ts

```diff
@@ -98,6 +98,31 @@ describe('Service Worker Schematic', () => {
     expect(tree.exists(path)).toEqual(true);
   });
 
+  it('should add $schema in ngsw-config.json with correct relative path', () => {
+    const pathToNgswConfigSchema = 'node_modules/@angular/service-worker/config/schema.json';
+
+    const name = 'foo';
+    const rootAppOptions: ApplicationOptions = {
+      ...appOptions,
+      name,
+      projectRoot: '',
+    };
+    const rootSWOptions: ServiceWorkerOptions = {
+      ...defaultOptions,
+      project: name,
+    };
+    const rootAppTree = schematicRunner.runSchematic('application', rootAppOptions, appTree);
+    const treeInRoot = schematicRunner.runSchematic('service-worker', rootSWOptions, rootAppTree);
+    const pkgTextInRoot = treeInRoot.readContent('/ngsw-config.json');
```

Comment:
`pkgText...` --> `configText...`

Reply:
I've kept the same names as in the other tests.

Reply:
It doesn't change the fact that it is a bad, misleading name 😛
I would change it in other tests too 😇